curl_cffi/aio.py

```diff
@@ -53,19 +121,22 @@ def timer_function(curlm, timeout_ms: int, clientp: Any):
 def socket_function(curl, sockfd: int, what: int, clientp: Any, data: Any):
     async_curl = ffi.from_handle(clientp)
     loop = async_curl.loop
+    selector = _get_selector(loop)
 
-    # Always remove and readd fd
-    if sockfd in async_curl._sockfds:
-        loop.remove_reader(sockfd)
-        loop.remove_writer(sockfd)
+    # # Always remove and readd fd
+    # if sockfd in async_curl._sockfds:
+    #     selector.remove_reader(sockfd)
+    #     selector.remove_writer(sockfd)
```

Comment:
Any particular reason why these lines are not needed any more?

Reply:
here is the error I got that caused by these lines I think:
```
unhandled exception during asyncio.run() shutdown
task: <Task finished name='Task-1' coro=<main() done, defined at C:\Users\User\Desktop\testenv12\proact.py:7> exception=KeyError(672)>
Traceback (most recent call last):
  File "C:\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Python310\lib\asyncio\base_events.py", line 633, in run_until_complete
    self.run_forever()
  File "C:\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Python310\lib\asyncio\base_events.py", line 600, in run_forever
    self._run_once()
  File "C:\Python310\lib\asyncio\base_events.py", line 1860, in _run_once
    event_list = self._selector.select(timeout)
  File "C:\Python310\lib\asyncio\windows_events.py", line 439, in select
    self._poll(timeout)
  File "C:\Python310\lib\asyncio\windows_events.py", line 788, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Tester\Desktop\testenv12\proact.py", line 9, in main
    r = await s.get(URL)
  File "C:\Python310\lib\site-packages\curl_cffi\requests\session.py", line 930, in request
    await task
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Tester\Desktop\testenv12\proact.py", line 8, in main
    async with requests.AsyncSession(impersonate="chrome101") as s:
  File "C:\Python310\lib\site-packages\curl_cffi\requests\session.py", line 799, in __aexit__
    self.close()
  File "C:\Python310\lib\site-packages\curl_cffi\requests\session.py", line 804, in close
    self.acurl.close()
  File "C:\Python310\lib\site-packages\curl_cffi\aio.py", line 170, in close
    self._selector.remove_reader(sockfd)
  File "C:\Python310\lib\site-packages\curl_cffi\_asyncio_selector.py", line 235, in remove_reader
    del self._readers[fd]
KeyError: 672
```

Reply:
Also notice added:
```py
        selector.remove_reader(sockfd)
        selector.remove_writer(sockfd)
```
at `if what & CURL_POLL_REMOVE:`